### PR TITLE
GVL: Added popup with error message for unsupported qr code format (P…

### PR DESCRIPTION
**Pull request by zladzeyka**
…IA-1553)

### Description
added popup for unsupported qr code

### How to test
https://ginis.atlassian.net/browse/PIA-1068

### Merging
Automatic

### CODE REVIEW

**Review by a-szotyori — APPROVED** [review]
I found a bug while testing and while at it I also implemented a fix for it. 😄

**Comment by a-szotyori** [comment]
We need to add color configuration for the red color.

**Review by zladzeyka — COMMENTED** [review]
Looks good!

File: GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift [hunk 1]
```diff
@@ -329,10 +329,10 @@ extension CameraViewController {
     
     fileprivate func configurePopupViewForUnsupportedQR(_ newQRCodePopup: QRCodeDetectedPopupView,
                                                         dismissCompletion: @escaping () -> Void) {
-        newQRCodePopup.qrText.textColor = .red
+        newQRCodePopup.qrText.textColor = giniConfiguration.unsupportedQrCodePopupColor
```
Comment (a-szotyori) [inline GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift:332]: In the `QRCodeDetectedPopupView` we are using a `GiniColor` and our `UIColor.from()` extension:
```
qrText.textColor = UIColor.from(giniColor: giniConfiguration.qrCodePopupTextColor)
``` 

We should do it the same here as well and then also rename it to `unsupportedQrCodePopupTextColor`.

```suggestion
        newQRCodePopup.qrText.textColor = UIColor.from(giniColor: giniConfiguration.unsupportedQrCodePopupTextColor)
```

File: GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift [hunk 2]
```diff
@@ -329,10 +329,10 @@ extension CameraViewController {
...
         newQRCodePopup.qrText.text = .localized(resource: CameraStrings.unsupportedQrCodeDetectedPopupMessage)
         newQRCodePopup.proceedButton.setTitle("✕", for: .normal)
-        newQRCodePopup.proceedButton.setTitleColor(.red, for: .normal)
+        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupColor, for: .normal)
```
Comment (a-szotyori) [inline GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift:335]: Also then we should add an `unsupportedQrCodePopupButtonColor` as well to be analogous with `qrCodePopupButtonColor`. And also add the same highlighted state color as for the normal popup.
```suggestion
        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor, for: .normal)
        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor.withAlphaComponent(0.5),
                                                   for: .highlighted)
```

**Review by a-szotyori — CHANGES_REQUESTED** [review]

File: GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift [hunk 3]
```diff
@@ -323,6 +327,18 @@ extension CameraViewController {
         }
     }
     
+    fileprivate func configurePopupViewForUnsupportedQR(_ newQRCodePopup: QRCodeDetectedPopupView,
+                                                        dismissCompletion: @escaping () -> Void) {
+        newQRCodePopup.backgroundColor = UIColor.from(giniColor:giniConfiguration.unsupportedQrCodePopupBackgroundColor)
+        newQRCodePopup.qrText.textColor = UIColor.from(giniColor: giniConfiguration.unsupportedQrCodePopupTextColor)
+        newQRCodePopup.qrText.text = .localized(resource: CameraStrings.unsupportedQrCodeDetectedPopupMessage)
+        newQRCodePopup.proceedButton.setTitle("✕", for: .normal)
+        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor, for: .normal)
```
Comment (a-szotyori) [inline GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift:336]: Also need to set the color for the highlighted state (same as the `QRCodeDetectedPopupView` does on line 73):
```suggestion
        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor, for: .normal)
newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor.withAlphaComponent(0.5),                                                   for: .highlighted)
```

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**